catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -126,7 +136,30 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 filterBuilder);
 
         Stopwatch stopwatch = Stopwatch.createStarted();
-        QueryResponse response = catalogFramework.query(request);
+        QueryResponse response = null;
+
+        if (configuration.isDisableLocalCatalog()) {
```

Comment:
❓ Since we only query for a single source at a time (https://github.com/codice/ddf/blob/9939ef34e50441223d9114ef4c320a97f840106c/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java#L131-L140), could this block be refactored into something like (if we refactor this chunk into a function):

```java
private Response query(<args>) {
    if (configuration.isDisableLocalCatalog() && request.sourceIds().contains(catalogFramework.getId()) {
        return new QueryResponseImpl(request, new ArrayList<>(), 0);
    }
    //otherwise do normal logic
    ...
}  
```